app/controllers/cloud_volume_controller.rb

```diff
@@ -101,18 +101,18 @@ def attach_volume
   def attach_finished
     task_id = session[:async][:params][:task_id]
     volume_id = session[:async][:params][:id]
-    volume_name = session[:async][:params][:name]
     vm_id = session[:async][:params][:vm_id]
     vm = find_record_with_rbac(VmCloud, vm_id)
+    volume = find_record_with_rbac(CloudVolume, volume_id)
```

Comment:
We click this option from cloud volume ui. So, i am assuming we never come across a scenario where record never exists. We almost have this line in every controller.

Reply:
It looks like it will raise a RecordNotFound exception rather than return nil: https://github.com/kavyanekkalapu/manageiq-ui-classic/blob/e74bb01a49fc80b8a676469ddcd092b8344f791e/app/controllers/mixins/checked_id_mixin.rb#L61